Resources/Prototypes/Catalog/Fills/Lockers/security.yml

```diff
@@ -179,10 +183,10 @@
     contents:
       - id: ClothingEyesGlassesSecurity
         prob: 0.3
-      - id: ClothingHeadHatFedoraBrown
-      - id: ClothingNeckTieDet
-      - id: ClothingOuterVestDetective
-      - id: ClothingOuterCoatDetective
+      - id: ClothingHeadHatBeretForensicSpecialist # DeltaV - replace ClothingHeadHatFedoraBrown in favour of a beret. 
```

Comment:
Can't remember the changelog but this needs a changelog entry for Detective having a full resprite

Reply:
Will update.